source/fundamentals/crud/run-command.txt

```diff
@@ -114,32 +125,46 @@ run the ``count`` command on the ``flowers`` collection of the
 Output
 ~~~~~~
 
-In the output, you should see the number of documents in
-the collection that match the query as the value of the ``n`` field, as
-well as the command execution information:
+In the output, you should see fields explaining the
+execution of the count operation, such as the winning plan and rejected
+plans. The output also contains the explain command execution information:
```

Comment:
Issue: I could see this being a little hard to parse, especially for someone not familiar with explain commands. For example, do we expect the reader to know what a "winning plan" is in this context?

I also initially had trouble parsing the last sentence. Maybe change to "The output also contains information about the execution of the ``explain`` command."

Reply:
I added a clause explaining what a winning plan is.